src/Component/Card-note/CardNote.jsx

```diff
@@ -0,0 +1,53 @@
+import { useNote } from "../../Context/Note/note-context";
+import { useState } from "react";
+import "../../Style/cardNote.css";
+import Typographycal from "../Typographcal/Typographycal";
+import Palette from "../Palette/Palette";
+import Tags from "../Tags/Tags";
+import Priority from "../Priority/Priority";
+import { SET_TITLE } from "../../Variable/variable";
+function CardNote() {
+  const { createNotes, noteDispatch, noteState } = useNote();
+
+  return (
+    <>
+      {/* note card component */}
+      <div className=" m-y-5  wt-50 m-auto">
+        <div className="box-shadow pd-5 flex flex-column  m-y-8 rounded-s position-rel">
+          <div className="flex justify-btw pd-x-4 ">
+            <input
+              type="text"
+              className="outline-none border-none text-lg pd-3"
+              placeholder="Title"
+              onChange={(e) =>
+                noteDispatch({ type: SET_TITLE, payload: e.target.value })
+              }
+            />
+            <button className="text-color-5 bg-black-0 rounded-full outline-none border-none pd-x-3 pd-y-2 cursor">
+              <i className="fa-solid fa-thumbtack"></i>
+            </button>
+          </div>
+          <Typographycal></Typographycal>
```

Comment:
You can use self closing tag like : <Typographycal />

Reply:
okk I forgot to do that